topic-operator/src/main/java/io/strimzi/operator/topic/BatchingTopicController.java

```diff
@@ -1117,24 +1131,89 @@ private static Collection<AlterConfigOp> buildAlterConfigOps(Reconciliation reco
         return alterConfigOps;
     }
 
+    /**
+     * <p>The Topic Operator {@code alterableTopicConfig} can be used to specify a comma separated list of Kafka
+     * topic configurations that can be altered by users through {@code .spec.config}. Keep in mind that if changes
+     * are applied directly in Kafka, the operator will try to revert them producing a warning.</p>
+     *
+     * <p>This is useful in standalone mode when you have a Kafka service that restricts alter operations
+     * to a subset of all the Kafka topic configurations.</p>
+     *
+     * <p>The default value is "all", which means no restrictions in changing {@code .spec.config}.
+     * The opposite is "none", which can be set to explicitly disable any change.</p>
+     *
+     * @param alterConfigOps Requested alter config operations.
+     */
+    private void skipNonAlterableConfigs(Set<AlterConfigOp> alterConfigOps) {
+        var alterableConfigs = config.alterableTopicConfig();
+        if (alterableConfigs != null && alterConfigOps != null && !alterableConfigs.isEmpty()) {
+            if (alterableConfigs.equalsIgnoreCase("none")) {
```

Comment:
despite there is an ignore case, you are mentioning "none" (lower case) here but "NONE" (upper case) in the previous doc. I would stick with the same on both, i.e. "none" lower case.

Reply:
Right, let's keep them aligned.

Reply:
Ok, so, "none" everywhere including the documentation?

Reply:
I am just for consistency so even the upper case is fine with me.

Reply:
@john-mcpeek Well, it makes no sense to have `none` and `ALL`. So one way or another there will be another change needed. It was actually @tombentley who suggested upper case in https://github.com/strimzi/strimzi-kafka-operator/pull/9606#issuecomment-2046648540 and I think his logic makes sense. So if Paolo doesn't care you should go with uppercase everywhere.

Reply:
Tom originally suggested to have all uppercase as it is more evident comparing to configuration properties which are lowercase. So let's use UPPERCASE.

Reply:
Ok, I think we are NONE and ALL everywhere now.